### PR TITLE
HoP -> XO

### PR DESCRIPTION
**Pull request by ike709**
:cl: ike709
tweak: The Head of Personnel has been replaced by the Executive Officer.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Either this PR or #605 will need to be tweaked depending on which is merged first.

**Comment by Vivalas** [comment]
Can you post screenshots of sprite changes?

**Comment by ike709** [comment]
@Vivalas Sprites aren't changed, just renamed.

File: _maps/map_files/Trailblazer/Trailblazer.dmm [hunk 1]
```diff
@@ -12519,11 +12519,14 @@
...
 /area/shuttle/ftl/turret_protected/ai)
 "Alp" = (
+<<<<<<< HEAD
 /obj/machinery/door/airlock/command{
 	name = "Executive Officer";
 	req_access = null;
 	req_access_txt = "30"
 	},
+=======
+>>>>>>> refs/remotes/FTL13/master
```
Comment (Vivalas) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: ahhhh conflict markers

File: _maps/map_files/Trailblazer/Trailblazer.dmm [hunk 2]
```diff
@@ -12519,11 +12519,14 @@
 /turf/open/floor/plasteel/circuit,
 /area/shuttle/ftl/turret_protected/ai)
 "Alp" = (
+<<<<<<< HEAD
```
Comment (Vivalas) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: ahhhhh 🔥

File: _maps/map_files/Trailblazer/Trailblazer.dmm (hunk 1 above)
Comment (Vivalas) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: ahh conflict markers

Comment (ike709) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: God damn it

**Pull request merged**